sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/tests/JObjectTests.cs

```diff
@@ -347,17 +336,30 @@ public void TestWebPubSubConnectionJsonSerialize()
         }
 
         [TestCase]
-        public void TestWebPubSubContext_UserEventStates()
+        public void TestWebPubSubContext_UserEventStates_Legacy()
         {
             WebPubSubConfigProvider.RegisterJsonConverter();
-            var states = new Dictionary<string, object>
-            {
-                { "aKey", "aValue"},
-                { "bKey", 123 },
-                { "cKey", new StateTestClass() }
-            };
-            var decodedStates = states.EncodeConnectionStates().DecodeConnectionStates();
-            var context = new WebPubSubConnectionContext(connectionId: "connectionId", userId: "userA", eventName: "connected", eventType: WebPubSubEventType.System, hub: null, states: decodedStates);
+            var states =
+                new Dictionary<string, object>
+                {
+                    { "aKey", "aValue"},
+                    { "bKey", 123 },
+                    { "cKey", new StateTestClass() }
+                }
+                .EncodeConnectionStates()
+                .DecodeConnectionStates();
```

Comment:
Nit - these may no longer be needed.  At one point they were added to force arbitrary objects to be serialized as JSON, but I think we might now be doing that automatically on this code path.